linux-pull-run.sh

```diff
@@ -89,8 +89,13 @@ if [[ ! $no_open ]]; then
 	fi
 fi
 
+# Create the Docker volume
+path=`pwd`
+mkdir -p recordedData
+docker volume create --name chasecar --opt type=none --opt device=${path}/recordedData --opt o=bind
+
 # Run the Docker image
-docker run -p 3000:3000 ghcr.io/badgerloop-software/chase-car-dashboard-image:$tag
+docker run -v chasecar:/chase-car-dashboard/Backend/src/recordedData ghcr.io/badgerloop-software/chase-car-dashboard-image:$tag
```

Comment:
```suggestion
docker run -p 3000:3000 -p 4001:4001 -v chasecar:/chase-car-dashboard/Backend/recordedData/processedData ghcr.io/badgerloop-software/chase-car-dashboard-image:$tag
```
Accidentally removed the 3000:3000 mapping. Also, there really wasn't any way for you to know this, but the data recording addition needs a 4001:4001 mapping to allow for HTTP requests between the frontend and backend.

Also, I would add a processedData directory in `Backend/recordedData/`, ignore its contents in `.gitignore`, and change `PROCESSED_DATA_PATH` in `api.js` to point to that new directory. That way, we'll have our raw and processed data in similar locations.
- Just a note about ignoring processedData's contents: You might need to add a README in `processedData/`, like is being done in `/Backend/recordedData/sessions/`, so that git tracks the directory. Then, ignore any .csv files in that directory in `.gitignore` (you can replace the `/Backend/src/routes/*.csv` line with this).